src/main/resources/modules/stroke_exm_105_r4.json

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
I ran this twice via `fhir-patient-generator` `EXM_105_r4` branch, and am getting No Population for all patients. Not immediately clear to me why, but are you guys getting patients in measure populations? @okeefm @mgramigna

Reply:
Did you delete the `synthea/` directory then re-run `make`?

Reply:
Also, I think there may need to be updates to that branch now that `fhir-bundle-calculator` has updated. I'll make those shortly.

Reply:
Good call, my bad I forgot `make clean` doesn't remove `synthea`. I got better results this time so 👍